tutorials/misc/plot_sensor_locations.py

```diff
@@ -0,0 +1,180 @@
+#     end.
+#
+# *Layouts* give sensor positions in 2 dimensions (defined by ``x``, ``y``,
+# ``width``, and ``height`` values for each sensor), and are primarily used for
+# illustrative purposes (i.e., making diagrams of approximate sensor
+# positions). In contrast, *montages* give sensor positions in 3D (``x``,
+# ``y``, ``z``, in meters). In addition, montages come in two "flavors" —
+# :class:`~mne.channels.Montage` objects (for idealized sensor montages) and
+# :class:`~mne.channels.DigMontage` (for subject-specific digitizations of
+# sensor positions).
+#
+#
+# Working with layout files
+# ~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# To load a layout file, use the :func:`mne.channels.read_layout`
+# function, and provide the filename *without* its file extension. You can then
+# visualize the layout using its :meth:`~mne.channels.Layout.plot` method, or
+# (equivalently) by passing it to :func:`mne.viz.plot_layout`:
+
+biosemi_layout = mne.channels.read_layout('biosemi')
+biosemi_layout.plot()  # same result as: mne.viz.plot_layout(biosemi_layout)
+
+###############################################################################
+# Similar to the ``picks`` argument for selecting channels from
+# :class:`~mne.io.Raw` objects, the :meth:`~mne.channels.Layout.plot` method of
+# :class:`~mne.channels.Layout` objects also has a ``picks`` argument. However,
+# because layouts only contain information about sensor name and location (not
+# sensor type), the :meth:`~mne.channels.Layout.plot` method only allows
+# picking channels by index (not by name or by type).
+
+midline = np.where([name.endswith('z') for name in biosemi_layout.names])[0]
+biosemi_layout.plot(picks=midline)
+
+###############################################################################
+# If you're working with a :class:`~mne.io.Raw` object that already has sensor
+# positions incorporated, you can create a :class:`~mne.channels.Layout` object
+# with either the :func:`mne.channels.make_eeg_layout` function or the
+# :func:`mne.channels.find_layout` function.
+
+layout_from_raw = mne.channels.make_eeg_layout(raw.info)
+# same result as: mne.channels.find_layout(raw.info, ch_type='eeg')
+layout_from_raw.plot()
+
+###############################################################################
+# .. note::
+#
+#     There is no corresponding ``make_meg_layout`` function because sensor
+#     locations are fixed in a MEG system (unlike in EEG, where the sensor caps
+#     deform to fit each subject's head). Thus for MEG layouts you can simply
+#     load them with :func:`mne.channels.read_layout` or
+#     :func:`mne.channels.find_layout` as shown above.
+#
+# All :class:`~mne.channels.Layout` objects have a
+# :meth:`~mne.channels.Layout.save` method that allows writing layouts to disk,
+# in either ``.lout`` or ``.lay`` format (which format is inferred from the
+# file extension you pass to the method's ``fname`` parameter).
```

Comment:
at this point, the reader does not know what to make of either .lout or .lay - perhaps a short intro above would be good